src/psyclone/psyir/transformations/omp_taskwait_trans.py

```diff
@@ -253,9 +251,9 @@ def get_forward_dependence(taskloop, root):
         if parent_parallel is None:
             fwriter = FortranWriter()
             raise InternalError(
-                    LazyString(lambda: "No parent parallel directive was "
-                                       "found for the taskloop region: {0}"
-                               .format(fwriter(taskloop).rstrip("\n"))))
+                    LazyString(lambda: f"No parent parallel directive was "
+                                       f"found for the taskloop region: "
+                                       f"{fwriter(taskloop).rstrip(chr(10))}"))
```

Comment:
Again, comment about `chr(10)` please (if it's not possible/wise to reinstate the `'\n'`?).